src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -332,7 +332,7 @@ public override IList<CustomAttributeData> GetCustomAttributesData()
             bool throwOnFileNotFound,
             IntPtr ptrLoadContextBinder = default)
         {
-            return InternalLoadAssemblyName(assemblyRef, reqAssembly, ref stackMark, IntPtr.Zero, true /*throwOnError*/, ptrLoadContextBinder);
+            return InternalLoadAssemblyName(assemblyRef, reqAssembly, ref stackMark, IntPtr.Zero, throwOnFileNotFound, ptrLoadContextBinder);
```

Comment:
This is not directly related to the PR, but it should be fixed nevertheless. The method takes the `throwOnFileNotFound` parameter, but it didn't actually use it.